### PR TITLE
Added a demo for notifications

### PR DESCRIPTION
**Pull request by RikHeijdens**

### CODE REVIEW

File: notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/DownloadImageTask.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+/**
+ * Downloads images and returns them as bitmap.
+ *
+ * Clients need to extend it and implement their own onPostExecute method.
+ */
+public abstract class DownloadImageTask extends AsyncTask<String, Void, Bitmap> {
+
+	@Override
+	protected Bitmap doInBackground(String... urls) {
```
Comment (9ae) [inline notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/DownloadImageTask.java:22]: It seems like we are only ever downloading the first url, and returning that is a bitmap. Is the correct?

File: notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/Sample.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		add(new PlaylistItem.Builder()
+				.file("http://content.jwplatform.com/manifests/6QhkxD1P.m3u8")
+				.image("http://assets-jpcust.jwpsrv.com/thumbs/6QhkxD1P-720.jpg")
+				.mediaId("6QhkxD1P")
+				.description("Caminandes 3: Llamigo (2016)")
+				.title("Caminandes 3: Llamigo")
+				.build());
+	}};
+
+	private Sample() {}
```
Comment (9ae) [inline notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/Sample.java:36]: When is this constructor accessed?

**Review by 9ae — APPROVED** [review]

File: notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/DownloadImageTask.java (hunk 1 above)
Comment (RikHeijdens) [reply notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/DownloadImageTask.java:22]: Correct. That's intended - it never needs to download more than a single image at a time.

File: notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/Sample.java (hunk 2 above)
Comment (RikHeijdens) [reply notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/Sample.java:36]: Making the constructor private makes it impossible to instantiate this class, so to answer your question: never.

**Pull request merged**